pkg/kubelet/cm/dra/plugin/plugins_store.go

```diff
@@ -23,70 +23,87 @@ import (
 	"sync"
 )
 
-// PluginsStore holds a list of DRA Plugins.
-type pluginsStore struct {
+// Store keeps track of how to reach plugins registered for DRA drivers.
+// Each plugin has a gRPC endpoint. There may be more than one plugin per driver.
+//
+// The null Store is usable.
+type Store struct {
 	sync.RWMutex
-	// plugin name -> Plugin in the order in which they got added
+	// driver-name -> Plugin in the order in which they got added
 	store map[string][]*Plugin
 }
 
-// draPlugins map keeps track of all registered DRA plugins on the node
-// and their corresponding sockets.
-var draPlugins = &pluginsStore{}
```

Comment:
This is now embedded in the `Manager` type -- great -- I have never liked that this was a global variable.